lib/model.js

```diff
@@ -959,6 +959,10 @@ Model.prototype.sync = function(options) {
     , attributes = this.tableAttributes;
 
   return Promise.try(function () {
+    if (options.hooks) {
```

Comment:
What's the default value of `options.hooks`? `undefined`? should probably be `true`